code/Coding_Abgabe_I/ro.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+<div class="ro"> 
+
+    <p>Lorem ipsum dolor sit amet, 
+        consetetur sadipscing elitr, 
+        sed diam nonumy eirmod tempor invidunt ut 
+        labore et dolore magna aliquyam erat, 
+        sed diam voluptua. At vero eos et accusam et</p>
+</div>
+
+<!How do I style this? I tried with this class but it didn't work>

```

Comment:
Here (and in the HTML-files ni.html and ju.html), I recommend only using `<div>`  instead of `<p>`, for example something like `<div class="information-ro">` and then style it in CSS.